Tt 190 - let user upload an profile image

NetanelSegal opened the pull request.

judele124 reviewed (changes requested):
in app and trip navBar make sure to desplay user.imgUrl || htttp//{user.name}

olso i get eroor in this branch when i click on TripActionsModalProps the userId._id is null

NetanelSegal commented:
> in app and trip navBar make sure to desplay user.imgUrl || htttp//{user.name}
> 
> olso i get eroor in this branch when i click on TripActionsModalProps the userId._id is null

good job for noticing the navbar issue, i fixed it.
the error you were getting is becouse we had in the participent array a user that didnt exist in the db so the populate didn't work, its because i deleted all our data again.
now you souldn't get this error
